sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobQueryDelimitedSerialization.java

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.models;
+
+/**
+ * Defines the input or output delimited (CSV) serialization for a blob quick query request.
+ */
+public class BlobQueryDelimitedSerialization extends
+    BlobQuerySerialization {
+
+    /* TODO(gapra) : Consider diff datatype in case future break. */
```

Comment:
I think this will continue to be char from what I can tell from the discussions we've had with the QQ team